web-client/src/modules/myRequests/containers/FindRequestsContainer.tsx

```diff
@@ -21,21 +21,24 @@ import {
   getStreetAddressFromProfile,
```

Comment:
Made a bunch of changes to get FindRequestsContainer to work which is no longer relevant.